core/unix/os.c

```diff
@@ -373,6 +373,7 @@ static int num_fd_add_pre_heap;
 static byte *app_brk_map;
 static byte *app_brk_cur;
 static byte *app_brk_end;
+static byte *app_brk_request;
```

Comment:
Explain in a comment what this is and what invariants hold: is it always between app_brk_map and app_brk_cur, and only differs from app_brk_cur b/c _cur is aligned forward to the page boundary?